README.md

```diff
@@ -1,38 +1,137 @@
+ * application-{secure-profile}.properties. - This will contain only the 
+ information required for secure connections.  E.g. the urls, key, and secret (etc.)
+ values to connect to the ESB. 
+ 
+# Development
+ 
+There are no explicit dependencies in SPE on either Docker or
+OpenShift.  If you supply the data volumes and the services required
+by SPE it should run fine in Docker on a laptop or from the command
+line or in an IDE.
+
+If there are problems that cause a test instance to end up in a loop
+where it fails and then auto-deploys again either cancel the
+deployment from the deployment configuration page or just reduce the
+number of pods on that page to 0.
+
+# Input and Output
+SPE gets data from the SPE application in the IBM ESB.  It also
+maintains a small disk file storing the most recent time that it requested
+grade information.
+
+## Logs
+
+Logs are available in the pods created by the cronjob.  The
+names all start with 'spe' and end with a time stamp. If the pod
+appears in the Applications/Pods list for the project you can get the
+log from the UI. Additional logs should be available using the "View
+Archive" tab on that page.  Logs may also be available via the command
+line or in Spunk, or at
+https://kibana.openshift.dsc.umich.edu/
+
+# OpenShift Considerations
+
+## Creating an SPE instance
+A new instance of code should be based on the deployment and build
+configuration of the existing instances.  It is unlikely that a new
+instance will be required.  Updating an existing instance is covered
+below.
+
+The disk volumes for the date persistence and for the OpenShift
+secrets need to be mounted explicitly.  This can be done in deployment
+configuration yaml or in the OpenShift UI.
+
+## Updating a SPE instance
+
+A SPE application instance might need to be updated for several reasons.
+
+- *schedule*: The repetion frequency of the job may need to change.
+  This is done by changing the value of intervalSeconds in the
+  application properties file.
+- *new image*: If there is a new build the image specification must be
+updated. It shouldn't be "latest" for a production version.  Modifying
+this requires editing the deployment configuration yaml.
+- *application arguments*:  The list of application property files used by the script can
+be modified by changing the container arguments in the deployment
+configuration. The list will be different for different instances.
+See the properties file section above for more details.
+
+# SPECIAL TASKS
+
+## Adjust the last retrieved date
+
+In rare circumstances it may be necessary to reset the last queried
+date used by SPE.  The file used to store the last retrieved date is a
+simple text file stored in the file *persisted.txt*.  In OpenShift
+this can be edited directly by adding a new app that just runs a bash
+shell to the project and then attaching the data volume holding the
+persisted data to that pod. The file can then be edited in vi.
```

Comment:
ReadMe seems detailed, but I don't see how to run SPE in OpenShift instruction probably best place would in the CtoolsWiki and provide a link here would a good idea